crates/cairo-lang-parser/src/parser.rs

```diff
@@ -637,6 +637,7 @@ impl<'a> Parser<'a> {
                     SyntaxKind::TerminalLBrace if lbrace_allowed == LbraceAllowed::Allow => {
                         Some(self.expect_constructor_call(path).into())
                     }
+                    SyntaxKind::TerminalNot => Some(self.expect_macro_call(path).into()),
```

Comment:
```suggestion
                    SyntaxKind::TerminalNot => Some(self.expect_macro_call(path).into()),
```

Should we rename `TerminalNot` to `TerminalBang`? We're overloading its semantics here a bit